packages/arb-token-bridge-ui/src/components/common/PendingTransactions.tsx

```diff
@@ -0,0 +1,73 @@
+import { motion } from 'framer-motion'
```

Comment:
Is this component reused anywhere outside of Transaction history? Otherwise I would add it in `TransactionsTable`

Reply:
Note: I mean in its own file, but in `TransactionsTable` folder